### PR TITLE
Adds Wagtail line to CMS README

### PR DESCRIPTION
**Pull request by emileighoutlaw**
cc @leahbannon

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ We welcome you to explore our FEC repositories, make suggestions, and contribute
 - `openFEC <https://github.com/18F/openfec>`_: betaFEC’s API
 - `openFEC-web-app <https://github.com/18f/openfec-web-app>`_: the betaFEC web app for exploring campaign finance data
 - `fec-style <https://github.com/18F/fec-style>`_: shared styles and user interface components
-- `fec-cms <https://github.com/18F/fec-cms>`_: the content management system (CMS) for betaFEC
+- `fec-cms <https://github.com/18F/fec-cms>`_: the content management system (CMS) for betaFEC. This project uses Wagtail, an open source CMS written in Python and built on the Django framework.
```
Comment (noahmanger) [inline README.rst]: Should we link to the Wagtail repo?

Comment (emileighoutlaw) [inline README.rst]: Good call. Which URL should I link to here, @noahmanger ?

Comment (noahmanger) [inline README.rst]: Looks like this is their main repo https://github.com/torchbox/wagtail

Comment (emileighoutlaw) [inline README.rst]: Added!

**Pull request merged**